### DISCUSSION TITLE
[Question] how to define my own module?

### DISCUSSION THREAD

**Discussion by emacsbliss** [abougouffa/minemacs#132] [Q&A] [answered]
couldn't find it anywhere in documentation, not sure if it's supported..

Basically I wan to be able to define my own modules (similar to Doom Emacs) in ~/.minemacs.d

**Comment by abougouffa** [comment, marked as answer]
Hello @emacsbliss ,

Well, there is no *formal* way of doing that. You can do it directly in your `~/.minemacs.d/config.el` or in a separate file (ex. `~/.minemacs.d/modules/my-module.el`) and then load it from `config.el` with:

```elisp
(load (concat minemacs-config-dir "modules/my-module.el"))
```

**Reply by emacsbliss** [reply]
ok, thanks!
